ci-operator/config/openshift/release/openshift-release-master__ocp-4.6-4.8.yaml

```diff
@@ -0,0 +1,32 @@
+releases:
+  initial:
+    release:
+      channel: stable
+      version: "4.6"
+  intermediate:
+    release:
+      channel: stable
+      version: "4.7"
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.8"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-46-stable-to-48-ci
```

Comment:
your naming is off - you need to be consistent with naming of other jobs.

variant name:

```
stable-4.8-upgrade-from-stable-4.6
```

test name

```
e2e-aws-upgrade
```

Reply:
Fixed (renamed to `e2e-aws-upgrade-paused` so that it won't be confused with standard 4.5 -> 4.6 -> 4.7 upgrades)